src/styles/styles/card.scss

```diff
@@ -0,0 +1,147 @@
+.catalog__content {
+  box-sizing: border-box;
+  display: grid;
+  grid-template-rows: auto;
+  grid-template-columns: 200px;
+
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px ;
+  margin: 0 auto;
+
+  @media (min-width: 488px) {
+    grid-template-columns: 200px 200px;
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: 200px 200px 200px;
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: 200px 200px 200px 200px;
+  }
+  }
+
+  .card {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 200px;
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+    border: 1px solid $border-color;
+    border-radius: 5px;
+    transition: 0.3s;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+
+    &__img {
+      margin: auto;
+      width: 160px;
+      height: 134px;
+      margin-bottom: 40px;
+    }
+
+    &__title {
+      margin: 0;
+      margin-bottom: 4px;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+      color: $main-color;
+      transition: 0.3s;
+    }
+
+    &:hover .card__title {
+      color: $card-title-hover;
+
+    }
+
+    &__code {
+      margin: 0;
+      margin-bottom: 16px;
+      font-size: 10px;
+      line-height: 14px;
+      color: $secondary-color;
+    }
+
+    &__reviews {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 24px;
+    }
+
+    &__review_point {
+      margin: 0;
+      font-size: 10px;
+      color: $main-color;
+    }
+
+    &__price {
+      display: flex;
+      justify-content: space-between;
+      margin: 0;
+      margin-bottom: 16px;
+    }
+
+    &__price-text {
+      margin: 0;
+      font-size: 12px;
+      line-height: 18px;
+      color: $secondary-color;
+    }
+
+    &__price-number {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $main-color;
+    }
+
+    &__buy-link {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      box-sizing: border-box;
+      border: 1px solid $button-color;
+      border-radius: 5px;
+      font-size: 14px;
+      font-weight: 700;
+      line-height: 16px;
+      text-transform: uppercase;
+      text-decoration: none;
+      text-align: center;
+      padding: 11px 5px;
+      width: 100%;
+      color: $white-color;
+      background-color: $button-color;
+      transition: 0.3s;
```

Comment:
also you use 300ms for transition in different places, I think it would be a good idea to keep this value in variable

don't forget to update this value-variable in your scss files